src/worker/ingestion/process-event.ts

```diff
@@ -25,13 +24,16 @@ import { DB } from '../../utils/db/db'
 import { KafkaProducerWrapper } from '../../utils/db/kafka-producer-wrapper'
 import { elementsToString, personInitialAndUTMProperties, sanitizeEventName, timeoutGuard } from '../../utils/db/utils'
 import { status } from '../../utils/status'
-import { castTimestampOrNow, filterIncrementProperties, UUID, UUIDT } from '../../utils/utils'
-import { ActionManager } from './action-manager'
+import { castTimestampOrNow, extractElements, filterIncrementProperties, UUID, UUIDT } from '../../utils/utils'
 import { PersonManager } from './person-manager'
 import { TeamManager } from './team-manager'
 
+export interface IEventX extends IEvent {
```

Comment:
What's does the X mean?

Reply:
Renamed to `IEventMatchable`. Though still kind of an ugly way of passing this data, I will rework it to something a bit nicer

Reply:
Removed this extended interface in favor of much nicer `EventProcessingResult`.